.github/workflows/preview.yml

```
@@ -0,0 +1,70 @@
+name: Preview
+
+on: pull_request
+
+jobs:
+  release:
+    # Prevents changesets action from creating a PR on forks
+    if: github.repository == 'open-wc/open-wc'
+    name: Preview
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+
+      - name: Setup Node.js 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+          cache: 'yarn'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install Dependencies
+        run: yarn --frozen-lockfile
+      
+      - name: Build Documentation Site
+        run: yarn site:build
+
+      - name: Build the preview URL from the branch name
+        run: echo ::set-output name=url::https://${GITHUB_HEAD_REF}--open-wc-org.netlify.app/
+        id: preview
+
+      - name: Publish to Netlify
+        env:
+          NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+          PREVIEW_URL: ${{ steps.preview.outputs.url }}
```

according to [this](https://github.com/modernweb-dev/rocket/blob/main/packages/core/src/helpers/index.js#L13) it's called deploy url? what is preview URL?

and it should also set a context I think

how about this?

```suggestion
          DEPLOY_URL: ${{ steps.preview.outputs.url }}
          CONTEXT: deploy-preview
```